client/web/src/repo/blob/codemirror/token-selection/keybindings.ts

```diff
@@ -0,0 +1,163 @@
+        },
+    },
+    {
+        key: 'Mod-ArrowLeft',
+        run(view) {
+            view.state.facet(historyFacet).goBack()
+            return true
+        },
+    },
+    {
+        key: 'ArrowLeft',
+        run(view) {
+            const position = positionAtCmPosition(view, view.state.selection.main.from)
+            const table = view.state.facet(syntaxHighlight)
+            const line = position.line
+            const occurrence = closestOccurrenceByCharacter(line, table, position, occurrence =>
+                occurrence.range.start.isSmaller(position)
+            )
+            if (occurrence) {
+                selectOccurrence(view, occurrence)
+            }
+            return true
+        },
+    },
+    {
+        key: 'ArrowRight',
+        run(view) {
+            const position = positionAtCmPosition(view, view.state.selection.main.from)
+            const table = view.state.facet(syntaxHighlight)
+            const line = position.line
+            const occurrence = closestOccurrenceByCharacter(line, table, position, occurrence =>
+                occurrence.range.start.isGreater(position)
+            )
+            if (occurrence) {
+                selectOccurrence(view, occurrence)
+            }
+            return true
+        },
+    },
+    {
+        key: 'ArrowDown',
+        run(view) {
+            const position = positionAtCmPosition(view, view.state.selection.main.from)
+            const table = view.state.facet(syntaxHighlight)
+            for (let line = position.line + 1; line < table.lineIndex.length; line++) {
+                const occurrence = closestOccurrenceByCharacter(line, table, position)
+                if (occurrence) {
+                    selectOccurrence(view, occurrence)
+                    return true
+                }
+            }
+            return true
+        },
+    },
+    {
+        key: 'ArrowUp',
+        run(view) {
+            const position = positionAtCmPosition(view, view.state.selection.main.from)
+            const table = view.state.facet(syntaxHighlight)
+            for (let line = position.line - 1; line >= 0; line--) {
+                const occurrence = closestOccurrenceByCharacter(line, table, position)
+                if (occurrence) {
+                    selectOccurrence(view, occurrence)
+                    return true
+                }
+            }
+            return true
+        },
+    },
+    {
+        key: 'Escape',
+        run(view) {
+            closeHover(view)
+            view.contentDOM.blur()
+            return true
+        },
+    },
+]
+
+const remappedKeyBindings: Record<string, string> = {
+    // Open definition in a new tab.
+    Enter: 'Mod-Enter',
+    // Vim bindings.
+    ArrowUp: 'k',
+    ArrowDown: 'j',
+    ArrowLeft: 'h',
+    ArrowRight: 'l',
+}
+
+export const tokenSelectionKeyBindings = keybindings.flatMap(keybinding => {
+    const remappedKey = remappedKeyBindings?.[keybinding.key ?? '']
```

Comment:
```suggestion
    const remappedKey = remappedKeyBindings[keybinding.key ?? '']
```

Can it just be this?

Reply:
This works 👍🏻 Thanks